woocommerce/woocommerce
# Update SSR version to check against major versions only closes #27972

### All Submissions:






### Changes proposed in this Pull Request:

Change the SSR "WC tested up to" to check for major versions instead of minor.



Closes #27972

### How to test the changes in this Pull Request:

1. Use an extension such as `woocommerce-bookings`. Open up the main file and in the header, change the "WC tested up to" to "3.0"
2. Go to the WC SSR page and scroll down to WooCommerce Bookings and ensure you see the message `Installed version not tested with active version of WooCommerce 4.0`
3. Now edit the header "WC tested up to" to "4.0".
4. Go back to the WC SSR page and scroll down to WooCommerce Bookings and ensure you don't see any message.

### Other information:




### Changelog entry

* Tweak - Test extension/plugins WC tested up to version against major WC versions instead of minor versions.

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
